### PR TITLE
Display help message in API store - Update Domains.

### PR DESCRIPTION
**Pull request by chamindias**
Fix for : https://wso2.org/jira/browse/APIMANAGER-3828

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
 		"replyAdded": "Reply Successfully Added",
 		"callBackHelpMsg": "This parameter can be used in the authorization code grant type when invoking an API. For example, assume that both the client and the API Gateway are run on the same server and the Authorization API URL is https://localhost:8243/authorize. <strong> query component:</strong> “response_type=code&client_id=<consumer_key>&scope=PRODUCTION&redirect_uri=<application_callback_url>” and <strong>headers:</strong> Content-Type:application/x-www-form-urlencoded",
 		"tokenHelpMsg" : "The period after which the token will expire after generation. A negative value ensures that the token never expires.",
-        "curlHelpMsg" : "CURL command created from the access tokens to send the request."
+        "curlHelpMsg" : "CURL command created from the access tokens to send the request.",
```
Comment (harsha89) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json]: slight formatting issue

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/item-design/js/api-design.js [hunk 2]
```diff
@@ -860,9 +860,8 @@ function updateContextPattern(){
     var version = $('#version').val();
 
     if(context != ""){
-        if(context.search("{") != -1 || context.search("}") != -1){
-            context = context.replace("{","");
-            context = context.replace("}","");
+        if( context.search("{}") != -1 ){
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/item-design/js/api-design.js]: formatting issue (space needed)

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/item-design/js/api-design.js [hunk 3]
```diff
@@ -828,9 +828,8 @@ function updateContextPattern(){
     var version = $('#version').val();
 
     if(context != ""){
-        if(context.search("{") != -1 || context.search("}") != -1){
-            context = context.replace("{","");
-            context = context.replace("}","");
+        if(context.search("{}") != -1 ){
+            context = context.replace("{}","");
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/item-design/js/api-design.js]: same formatting issue here

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js [hunk 4]
```diff
@@ -236,6 +236,22 @@ $(document).ready(function () {
...
+    });
+
+    $('.update_domain_popup_sand').click(function(){
+        $('#update_help_sand').toggle('fast', function()
+        {
+            $('#update_help_sand').html(i18n.t('info.domainUpdateHelp'));
+        });
+        return false;
+    });
+
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js]: Please fix the formatting issues in the above section (line 239, 247, 249)

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/responsive/templates/subscription/subscription-list/js/subscription-list.js [hunk 5]
```diff
@@ -220,6 +220,22 @@ $(document).ready(function () {
 	    return false;
 	 });
 
+    $('.update_domain_popup_prod').click(function(){
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/responsive/templates/subscription/subscription-list/js/subscription-list.js]: formatting issues

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json (hunk 1 above)
Comment (chamindias) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json]: Fixed in : https://github.com/wso2/carbon-apimgt/pull/1319

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/item-design/js/api-design.js (hunk 2 above)
Comment (chamindias) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/item-design/js/api-design.js]: Fixed in : https://github.com/wso2/carbon-apimgt/pull/1319

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/item-design/js/api-design.js (hunk 3 above)
Comment (chamindias) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/item-design/js/api-design.js]: Fixed in : https://github.com/wso2/carbon-apimgt/pull/1319

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js (hunk 4 above)
Comment (chamindias) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js]: Fixed in : https://github.com/wso2/carbon-apimgt/pull/1319

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/responsive/templates/subscription/subscription-list/js/subscription-list.js (hunk 5 above)
Comment (chamindias) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/responsive/templates/subscription/subscription-list/js/subscription-list.js]: Fixed in : https://github.com/wso2/carbon-apimgt/pull/1319

**Pull request merged**